Spinner/Spinner.module.less

```diff
@@ -0,0 +1,196 @@
+// Spinner.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/variables.less";
+@import "../styles/skin.less";
+
+@keyframes spinner {
+	0% {
+		opacity: 1;
+	}
+	100% {
+		opacity: 0;
+	}
+}
+
+.spinner {
+	.bg {
+		display: inline-block;
+		width: @agate-spinner-size;
+		height: @agate-spinner-size;
+		position: relative;
+
+		.fan1,
```

Comment:
We can simplify several of these repetitive sections if the "blades" have two classes, or we use the next sibling selector `+`. E.g.: 
```
.blade {}
.blade + .blade {}
.blade + .blade + .blade {}
// Alternatively
.blade:nth-child(1) {}
.blade:nth-child(2) {}
.blade:nth-child(3) {}
```